acceptance-tests/apps/postgresqlapp/internal/app/app.go

```diff
@@ -69,11 +76,9 @@ func fail(w http.ResponseWriter, code int, format string, a ...any) {
 }
 
 func schemaName(r *http.Request) (string, error) {
-	schema, ok := mux.Vars(r)["schema"]
+	schema := chi.URLParam(r, "schema")
 
 	switch {
-	case !ok:
-		return "", fmt.Errorf("schema missing")
 	case len(schema) > 50:
 		return "", fmt.Errorf("schema name too long")
 	case len(schema) == 0:
```

Comment:
**thought**: This is also the case when schema wasn't provided. We might re-word this as "schema name is required".

Reply:
updated